__tests__/api/health.spec.js

```diff
@@ -16,7 +16,11 @@
 //
 
 import { default as request } from 'supertest'; // eslint-disable-line
-import app from '../../src';
+import expressApp from '../../src';
```

Comment:
I don't think this change is need any more. Just `import 'import app from '../../src';`

Reply:
Fixed